go.mod

```diff
@@ -3,12 +3,11 @@ module github.com/cri-o/ocicni
 go 1.12
 
 require (
-	github.com/containernetworking/cni v1.0.0-rc1
-	github.com/containernetworking/plugins v0.8.7
-	github.com/fsnotify/fsnotify v1.4.9
-	github.com/onsi/ginkgo v1.13.0
-	github.com/onsi/gomega v1.10.1
-	github.com/sirupsen/logrus v1.4.2
-	github.com/vishvananda/netlink v1.1.0
-	golang.org/x/net v0.0.0-20210226172049-e18ecbb05110 // indirect
+	github.com/containernetworking/cni v1.0.1
+	github.com/containernetworking/plugins v1.0.1
+	github.com/fsnotify/fsnotify v1.5.1
+	github.com/onsi/ginkgo v1.16.4
+	github.com/onsi/gomega v1.16.0
+	github.com/sirupsen/logrus v1.8.1
+	github.com/vishvananda/netlink v1.1.1-0.20210811191823-e1a867c6b452
```

Comment:
who is pulling this one?

Reply:
I see https://github.com/containernetworking/plugins/blob/189d0c06aa6da0c4f052d3831b091e6ea79e6675/go.mod#L22